app/scss/components/service_status.scss

```diff
@@ -98,20 +102,142 @@
       }
     }
 
-    h2 {
-      font-size: 25px;
+    .section-title {
       font-weight: normal;
     }
 
-    h3 {
-      font-size: 16px;
-      font-weight: bold;
+    td, th {
+      font-size: $smallFont;
     }
+  }
+}
 
-    td, th {
-      font-size: 12px;
+// todo: refactor base.css so we no longer need this convoluted selector
+.page-container .card .service-status-container h2.service-status-title {
+  align-items: center;
+  color: $blue;
+  display: flex;
+  font-size: $h2Font;
+  justify-content: flex-start;
+  margin: 0;
+}
+
+.service-status-back-to-top {
+  text-align: right;
+}
+
+.service-status-back-to-top-link {
+  color: $blue;
+  font-size: $backToTopFont;
+  padding: .5rem;
+
+  &:focus,
+  &:hover {
+    outline: 3px solid $blue;
+  }
+}
+
+.service {
+  margin: 2rem 0;
+}
+
+.service-status-title-actions {
+  display: flex;
+  margin-left: 1rem;
+}
+
+.service-status-graph-container {
+  margin-top: 3rem;
+}
+
+.service-status-graph-title {
+  align-items: center;
+  display: flex;
+  font-size: $h2Font;
+  font-weight: 700;
+  justify-content: space-between;
+}
+
+.service-status-entities-table {
+  &:not(:first-of-type) {
+    margin-top: 2rem;
+  }
+
+  tr:first-of-type {
+    border-top: 0;
+  }
+}
+
+.service-status-section-title {
+  font-size: $h2Font;
+  margin: 2rem 0 1.25rem 0;
+}
+
+.service-status-entities-table-title {
+  align-items: center;
+  border-bottom: 3px solid $lighter-grey;
+  color: $dark-grey;
+  display: flex;
+  font-size: $bigFont;
+  font-weight: bold;
+  justify-content: space-between;
+  margin-bottom: 10px;
+  padding-bottom: 5px;
+  text-transform: none;
+}
+
+.button[for^="add-for-test"],
+.button[for^="add-for-production"] {
+  font-size: $normalFont;
+  line-height: 26px;
+  margin-right: 0;
+  text-transform: none;
+}
```

Comment:
I think this is no longer used as the buttons became links, and the for attributes where amended with the service name.

Reply:
right you are! 👍